### ISSUE TITLE
NoSpaceAfter reported on `-INF` const with PSR12 and Squiz rulesets

### ISSUE THREAD

**Issue by p4veI** [squizlabs/PHP_CodeSniffer#3695]
**Describe the bug**
I'm using the `INF` const in our codebase along with a case for `-INF`, this triggers rule `Squiz.WhiteSpace.OperatorSpacing.NoSpaceAfter` on the following example, which is unexpected.

**Code sample**
```php
return match ($value) {
        'Infinity' => INF,
        '-Infinity' => -INF,
        default => (float) $value,
};
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```
ERROR | [x] Expected at least 1 space after "-"; 0 found (PSR12.Operators.OperatorSpacing.NoSpaceAfter)
ERROR | [x] Expected 1 space after "-"; 0 found (Squiz.WhiteSpace.OperatorSpacing.NoSpaceAfter)
```

**Expected behavior**
space after `-` is not required

**Versions (please complete the following information):**
 - PHP: 8.1
 - PHPCS: 3.7.1
 - Standard: Slevomat, PSR12, Squiz

**Additional context**
this also conflicts with `SlevomatCodingStandard.Operators.NegationOperatorSpacing.InvalidSpaceAfterMinus` that correctly recognises INF as a number

**Comment by jrfnl** [comment]
Duplicate. This was already fixed by #3653 which will be included in the PHPCS 3.7.2 release.

**Comment by jrfnl** [comment]
P.S.: the issue has nothing to do with `INF`, but everything with `match` constructs.

**Comment by p4veI** [comment]
my bad, thx!

**Comment by jrfnl** [comment]
I suggest you close the issue as it has already been fixed.
